wiki/Beatmap_Editor_Guides/Adding_Custom_Hitsounds/de.md

```diff
@@ -0,0 +1,50 @@
+Eigene Hinsounds hinzufügen
+===========================
+
+Von dem Thread: [*Wie man einen Eigenen Hitsound hinzufügt.* von: neonat](https://osu.ppy.sh/forum/p/3215699)
+
+Eigene Hitsounds
+---------------------------
+
+Als erstes werden einige Hitsounds benötigt.
+
+-   Man kann sie aus der [Spieler-Hitsound-Bibliothek](Custom_Hitsound_Library "wikilink") bekommen.
```

Comment:
change `[Spieler-Hitsound-Bibliothek](Custom_Hitsound_Library "wikilink")` to ` [Spieler-Hitsound-Bibliothek](../Custom_Hitsound_Library)`